cna/docs/org-chart.md

```diff
@@ -0,0 +1,46 @@
+# Organizational Chart of Cloud Native Austria
+
+Cloud Native Austria is structured to efficiently oversee and manage activities, ensuring a collaborative and productive environment within the Austrian cloud-native community. 
+Below is the current organizational chart of the association.
+
+## Board of Directors
+
+The Board of Directors provides strategic leadership and ensures the association adheres to its mission and vision.
+
+- **Chairman**: Thomas Schütz
+  - Leads the association and oversees all operational aspects.
+- **Secretary**: Daniel Drack
+  - Handles all administrative duties including meeting minutes and correspondence.
+- **Treasurer**: Erik Auer
+  - Manages the finances of the association, including budgeting and financial reporting.
+
+## Deputy Roles
+
+Deputy roles are designed to assist and step in for their primary counterparts as needed.
+
+- **Deputy Chairman**: vacant
+  - Assists the Chairman and substitutes when the Chairman is unavailable.
+- **Deputy Secretary**: vacant
+  - Supports the Secretary in all administrative tasks and acts as a substitute when required.
+- **Deputy Treasurer**: vacant
+  - Aids the Treasurer in managing the association's financial tasks and assumes responsibilities in their absence.
+
+## Auditors
+
+Auditors are responsible for reviewing the association's accounts to ensure accuracy and compliance with legal standards.
+
+- **Auditor 1**: Johannes Grumböck
+- **Auditor 2**: Thomas Stagl
+
+## Friends of Cloud Native Austria
+
+- TBD
+
+## General Members
+
+Members are crucial as they contribute to the diversity and vitality of the association.
+
+| Vorname   | Nachname |
+| --------  | ------- |
+| Hello     | World    |
```

Comment:
I would not write palceholders here. I would delete it, or very soon you can write, Constanze and Andi at this table.

Anyway we should definitely ask every member if we can put there name on this list before we do so, as it is definitely GDPR relevant.